stable/velero/templates/deployment.yaml

```diff
@@ -18,6 +18,7 @@ spec:
   template:
     metadata:
       labels:
+        name: velero
         app.kubernetes.io/name: {{ include "velero.name" . }}
```

Comment:
I think a better solution would be to have this be `{[ include "velero.name" . }}-server` and `{[ include "velero.name" . }}-restic` for the Restic Pods

Reply:
I'm trying to make the pod labels consistent with the restic deamonset pods in https://github.com/helm/charts/blob/0c22c90a5d6756b8fab8804492fb1d912ee09b2b/stable/velero/templates/restic-daemonset.yaml#L19 and by doing that making sure the https://github.com/helm/charts/blob/0c22c90a5d6756b8fab8804492fb1d912ee09b2b/stable/velero/templates/service.yaml#L18 service is targeting the velero pods since the restic pods don't have a metrics endpoint :)

I can change the names to velero-server and velero-restic but i then need to change a lot more references, didn't wan't to introduce more changes for this simple addition.

Reply:
Ah I see, it's just a little confusing to have `name` and `app.kubernetes.io/name`, but given this already exists in the restic deployment, I'm ok with accepting this.